src/PlusCommon/PlusCommon.h

```diff
@@ -47,13 +47,13 @@ enum PlusImagingMode
 
 /* Define case insensitive string compare for Windows. */
 #if defined( _WIN32 ) && !defined(__CYGWIN__)
-#  if defined(__BORLANDC__)
-#    define STRCASECMP stricmp
-#  else
-#    define STRCASECMP _stricmp
-#  endif
+  #if defined(__BORLANDC__)
+    #define STRCASECMP stricmp

```

Comment:
Keep only one way of case-insensitive string compare. I'm OK with the new method but then remove the STRCASECMP macro.